components/brave_shields/core/common/brave_shields_settings_values.h

```diff
@@ -22,6 +22,12 @@ enum ControlType {
   DEFAULT,
 };
 
+enum AutoShredType {
```

Comment:
Q: Can we use the enum directly from the .mojom file? If not, we should add a comment stating that changes to this enum must be mirrored in the .mojom file, and vice versa.